dream_layer_backend/dream_layer.py

```
@@ -567,6 +567,360 @@ def get_controlnet_models_endpoint():
+            "status": "error",
+            "message": str(e)
+        }), 500
+
+@app.route('/api/preview-grid', methods=['POST'])
+def preview_grid():
+    """Generate a preview of the grid layout"""
+    try:
+        from dream_layer_backend_utils.labeled_grid_exporter import collect_images, GridTemplate
+        import tempfile
+        import base64
+        from io import BytesIO
```

Ruff (0.12.2)

`io.BytesIO` imported but unused

Remove unused import: `io.BytesIO`

(F401)